docs/release-notes.md

```diff
@@ -7,6 +7,32 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## 1.6.0 for Magento Commerce 2.4.2
+
+-  {:.new}<!-- Issue 558 -->**New Page Builder styling**. Page Builder made massive improvements to how it styles content. The changes now make it easy to create simple CSS selectors to override Page Builder's existing content type styles. These changes make it possible to create Page Builder themes with rich responsiveness for all content types.
+
+-  {:.new}<!-- Issue 429 -->**Row container now optional**. You can now create content in Page Builder without requiring a Row container. In addition to the Row, the following content types can now be dragged and dropped directly on the stage: HTML, Block, Dynamic Block, Columns, and Tabs.
+
+-  {:.new}<!-- Issue 636 -->**New responsive viewport switcher**. You can now preview your Page Builder content at different device widths (breakpoints) using the new Admin viewport switcher buttons above the stage. The Admin viewport switcher simulates how your content will be styled when displayed on the storefront using different devices.
+
+-  {:.new}<!-- Issue 637 -->**New breakpoint-specific form fields**. Content type field values can now be set to specific viewport breakpoints. Icon indicators next to the fields show which viewport the content type's field value is applied to.
+
+-  {:.new}<!-- Issue 559 -->**Removed predefined form field entries**. Advanced form settings for margins, paddings, and border-related properties (border, border width, and border radius) are now set as `default`, so that values can be defined by a module's CSS.
+
+-  {:.new}<!-- Issue 635, 557,  -->**Added stage padding**. Rows and Columns now provide padding around contained content types on the stage, not on the storefront. The stage padding makes it easier to access the mouseover option menus for both the container and contained content types.
```

Comment:
We named it "affordance". Maybe, make sense to rename it here as well?

Reply:
Done.